roles/network_plugin/cilium/templates/cilium-config.yml.j2

```diff
@@ -145,3 +145,13 @@ data:
 
   native-routing-cidr: "{{ cilium_native_routing_cidr }}"
   auto-direct-node-routes: "{{ cilium_auto_direct_node_routes }}"
+
+  # Hubble settings
+{% if cilium_enable_hubble %}
+  enable-hubble: "true"
+  hubble-metrics: "{{ cilium_hubble_metrics }}"
+  hubble-listen-address: ":4244"
+{% if cilium_enable_hubble_metrics %}
+  hubble-metrics-server: ":9091"
```

Comment:
Do we really need that ? isn't it already the default value ?

Reply:
I just checked and without this line Cilium does not listen on this port (and 9091 is the default value on the helm chart https://github.com/cilium/cilium/blob/aa42034f09a5ac1218a8c6dfe0ee0b9bcf6d4801/install/kubernetes/cilium/values.yaml#L517).

Reply:
ok so https://github.com/kubernetes-sigs/kubespray/blob/72136ae776930f0775c5c9f6b5580e4a3e9402f4/roles/network_plugin/cilium/templates/cilium-ds.yml.j2#L97 was useless before this PR :/

Reply:
Yes pretty much :/